RiscVPlatformPkg/Universal/Sec/SecMain.c

```diff
@@ -458,6 +494,35 @@ static VOID EFIAPI PeiCore(VOID)
   //
   (*PeiCoreEntryPoint) (&SecCoreData, (EFI_PEI_PPI_DESCRIPTOR *)&mPrivateDispatchTable);
 }
+
+/**
+  RISC-V SEC to PEI handler
+
+  We can't directly transition to PEI because sbi_init() doesn't pass the arguments that the PEI core entry point expects.

```

Comment:
Actually because we call our own `PeiCore` function and not the real PEI Core directly, we can `sbi_init` to this. `RiscVSecOpenSbiHandler` shouldn't be needed.

Reply:
seems so, if we dont need HartId information passed from sbi_hart_switch_mode()

Reply:
Done